exercises/simple-linked-list/src/test/scala/SimpleLinkedListTest.scala

```diff
@@ -0,0 +1,56 @@
+import org.scalatest.{FlatSpec, Matchers}
+
+class SimpleLinkedListTest extends FlatSpec with Matchers {
+
+  it should "handle single item list" in {
+    val list = SimpleLinkedList().add(1)
+    list.isEmpty should be (false)
+    list.value should be (1)
+  }
+
+  it should "handle single item has no next item" in {
```

Comment:
I think we could give property based testing a shot.. At the very least it might give some exercise exercise implementers exposure to another way of testing. They may/may not like property based testing. But, the implementers just need to run the tests - not implement them.

Will property based tests be difficult to generate from the canonical data?